source/includes/examples/_cart-pole.html.md

```diff
@@ -1,14 +1,12 @@
-## Example: Cart Pole
+# Cart Pole Example
```

Comment:
Will the top nav have "Examples" highlighted. If so does it make sense to append "Example" to each title... I kind of think it might still make sense since no one really knows what a "cart pole" or "mountain car" is.